frontend/src/concepts/pipelines/content/ViewPipelineServerModal.tsx

```diff
@@ -78,21 +78,19 @@ const ViewPipelineServerModal: React.FC<ViewPipelineServerModalProps> = ({
                   </DescriptionListDescription>
                 </DescriptionListGroup>
                 <DescriptionListGroup>
-                  <DescriptionListTerm>AWS_ACCESS_KEY_ID</DescriptionListTerm>
+                  <DescriptionListTerm>Access-Key</DescriptionListTerm>
                   <DescriptionListDescription>
-                    {objectStorageRecord?.AWS_ACCESS_KEY_ID}
+                    {objectStorageRecord?.['Access-Key']}
```

Comment:
dont access the keys like this `['Access-Key']`

use the enum: `AWS_KEYS.ACCESS_KEY_ID`